src/types/ScholarshipAmount.ts

```diff
@@ -108,6 +108,24 @@ namespace ScholarshipAmount {
     return { min, max, type };
   }
 
+  // Translates 'amount' to the format it should be displayed as
+  export function fromStorage(amount: ScholarshipAmount): ScholarshipAmount {
```

Comment:
I don't exactly recall off the top of my head why I decided not to include this method when I added the other one. I'm guessing it has to do something with the fact that storing it again with the weird values should be ok and `toString` should be able to handle the weird values. Though I guess certain elements need the raw values and the UI would need to sort of do the same kind of handling this method does.

Reply:
We should add test cases for `toStorage` and this one 🤔.